### ISSUE TITLE
build maiko fails -DRELEASE=201

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#929]
I thought I would try to compile for release 201 (following inc/versions.h) and I edited makefile-linux.x86_64-x changing 351 to 201. 
```
o ../linux.x86_64-x/rplcons.o
../src/rplcons.c:47:10: warning: implicit declaration of function 'POINTER_PAGE' is invalid in C99 [-Wimplicit-function-declaration]
  page = POINTER_PAGE(list);
         ^
../src/rplcons.c:51:5: warning: implicit declaration of function 'GCLOOKUP' is invalid in C99 [-Wimplicit-function-declaration]
    GCLOOKUP(item, ADDREF);
    ^
../src/rplcons.c:51:20: error: use of undeclared identifier 'ADDREF'
    GCLOOKUP(item, ADDREF);
                   ^
../src/rplcons.c:52:25: error: use of undeclared identifier 'DELREF'
    GCLOOKUP(cdr(list), DELREF);
                        ^
2 warnings and 2 errors generated.
```
same error with or without cast-align. 

You said you had done this not long ago, so I'm wondering if I'm doing something wrong.

**Comment by nbriggs** [comment]
Couple of missing include files when it's version 201 because it's NOT BIGVM and NOT NEWCDRCODING.
I've pushed a branch named build-201 which will compile (at least it does for me).

If you want to build sysouts you'll need a version 201 sysout to start from...

DON'T merge this branch.

**Comment by nbriggs** [comment]
p.s., you'll still need to change the RELEASE for Linux, I only included 201 in the darwin versions I built on my mac.
(luckily, I've managed to tether my phone to my mac, have a minimal signal, and have 20 minutes to spare after lunch)

**Comment by masinter** [comment]
Alas, no response to keyboard or mouse running on linux pi to Windows/WSL.At least I can try Loops on Hemlock and see if it runs Truckin.

**Comment by nbriggs** [comment]
Yeah, same on my mac.  I had did get the screen up, though, and the cursor blinking -- but any input or mouse click caused it to go away.   It will take more debugging than I can do here.

You might want to clone and rebuild on hemlock -- the /usr/local/ versions there are not up to date (as far as I know).
I have never seen different behavior on hemlock vs my mac -- I seriously doubt that's the problem with Loops/Truckin.  It's coming up with unnamed players (when it creates them and says NIL as the name) it's going to fail when it tries to use NIL as the object name for the target of any operations.

**Comment by masinter** [comment]
see #260 and https://github.com/masinter/homedir/blob/master/LOADLOOPS.DRIBBLE

@skaisler1

**Comment by nbriggs** [comment]
Back to fixing this -- after getting the include files correct one can do some things but then it dies at
```
    frame #5: 0x000000010b707925 ldex`error(cp=<unavailable>) at common.c:99:43 [opt]
    frame #6: 0x000000010b7502b4 ldex`htfind(ptr=3090432, casep=<unavailable>) at gchtfind.c:359:5 [opt]
    frame #7: 0x000000010b707f3a ldex`N_OP_cons(cons_car=<unavailable>, cons_cdr=<unavailable>) at conspage.c:425:3 [opt]
    frame #8: 0x000000010b70bdd6 ldex`dispatch at xc.c:385:7 [opt]
    frame #9: 0x000000010b733bc1 ldex`start_lisp at main.c:790:3 [opt]
    frame #10: 0x000000010b73373b ldex`main(argc=8, argv=<unavailable>) at main.c:737:3 [opt]
```
complaining that it's trying to decrement a 0 reference count.  This is in code that had compilation warnings about mismatched integer sizes for pointer to integer - so that's the place to start looking

**Comment by pamoroso** [comment]
Does the issue still occur?

**Comment by nbriggs** [comment]
Probably still happening- I’ve not worked on this recently
